resource/css/styles.css

```diff
@@ -5,6 +5,10 @@ li, td > a {
   font-size: 14px;
 }
 
+label {
```

Comment:
Global setting - this will affect all labels, in all places. See #799 for more information.